### DISCUSSION TITLE
True wireless

### DISCUSSION THREAD

**Discussion by Keeze** [alf45tar/PedalinoMini#430] [Show and tell]
Just finished my build of the Pedalino. I added an 5000 mAh powerbak, which should last for about 30 hours (I measured the current at about 180 mA when in use). I also replaced the Arduino pro mini for a teensy LC, cause the teensy is much better in USB coms and USB MIDI is standard implemented.

I used stained spruce and 3D printed sides and partly the back. @alf45tar thanks for this project!

![KCPedalino-front](https://user-images.githubusercontent.com/6306958/218185050-52533cb4-c31e-48e1-9e49-27c15cf6d23b.jpg)

![KCPedalino-back](https://user-images.githubusercontent.com/6306958/218185076-e40f8f74-0ab9-4896-a6ec-e046c6635279.jpg)

Cheers,
Kees

**Comment by alf45tar** [comment]
Very good job. You find your way to improve it.

**Comment by Keeze** [comment]
So I got a new amp... A Katana and had a spare RPi 3B+ laying around. So I built a BLE to USB midi hub and now I can control my Katana. It worked on the first try! Did I already tell how awesome this project is!

![IMG_20230622_235031](https://github.com/alf45tar/PedalinoMini/assets/6306958/7555d81e-e15f-4fdb-a05f-f1be3738fbf3)

![IMG_20230622_234918](https://github.com/alf45tar/PedalinoMini/assets/6306958/b56dbad9-c173-4fb8-858c-d994b6d8a51b)

**Reply by Celticpure** [reply]
Can your Pedalino Display change when you press a Channel No. on the Katana amp itself? So in essence a two way midi traffic situation.

**Reply by alf45tar** [reply]
PedalinoMini display every MIDI message received from any interface.
Not sure to understand your question.

**Reply by Celticpure** [reply]
If a manual Channel (patch) press on the katana amp was pressed, is this change in channel (patch) automatically shown on the Pedalino.

**Reply by alf45tar** [reply]
Is Katana able to send MIDI out? Can you see the MIDI message sent by the Katana to PedalinoMini display?

**Reply by Keeze** [reply]
I tried, but no, no message is display on the screen. Also, when I connect the Katana to a PC I get 3 Midi devices, but none of them appear to send anything. Only when I open Tone Studio for Katana (Boss app to configure the Katana) I see a bunch of sysex messages appear in the MidiOX screen when I press a Channel button....

**Reply by alf45tar** [reply]
It looks like Katana has its own language.
